tests/unittests/GraphTest.cpp

```diff
@@ -183,6 +183,39 @@ TEST(Graph, float16Conv) {
   EXPECT_EQ(convInst->getBias()->getElementType(), ElemKind::Float16Ty);
 }
 
+/// Check that we can create conv3D with float16.
+TEST(Graph, float16Conv3D) {
```

Comment:
I was following the example of `float16Conv` but I think it does add some value because instantiating a conv3d with float16 inputs isn't tested elsewhere